dtmi/innodisk/innoage/devcmd-1.json

```diff
@@ -0,0 +1,102 @@
+{
+    "@context": [
+        "dtmi:iotcentral:context;2",
+        "dtmi:dtdl:context;2"
+    ],
+    "@id": "dtmi:innodisk:innoage:devcmd;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Command",
+            "displayName": {
+                "en": "IsDeviceLife"
+            },
+            "name": "IsDeviceLife"
+        },
+        {
+            "@type": "Command",
+            "displayName": {
+                "en": "RstartComputer"
+            },
+            "name": "ResetPower"
+        },
+        {
+            "@type": "Command",
+            "displayName": {
+                "en": "Getversion"
+            },
+            "name": "Getversion"
+        },
+        {
+            "@type": "Command",
+            "displayName": {
+                "en": "Information"
+            },
+            "name": "azsphereInfo"
```

Comment:
Is this related to Azure Sphere? Should the name match the displayName?

Reply:
Yes, I will modify it.